Harrastuspassi/Controllers/Home/List/HomeViewController.swift

```diff
@@ -94,16 +100,30 @@ class HomeViewController: UIViewController, UITableViewDelegate, UIScrollViewDel
     }
     
     func fetchUrl(urlString: String) {
-        let config = URLSessionConfiguration.default;
-        let session = URLSession(configuration: config);
-        var url: URL?;
-        url = applyQueryParamsToUrl(urlString);
-        print(url);
-        let task = session.dataTask(with: url!, completionHandler: self.doneFetching);
-        task.resume();
+        if !isFetching {
+            let config = URLSessionConfiguration.default;
+            let session = URLSession(configuration: config);
+            var url: URL?;
+            url = applyQueryParamsToUrl(urlString);
+            print(url);
+            let task = session.dataTask(with: url!, completionHandler: self.doneFetching);
+            task.resume();
+        }
+    }
+    
+    func fetchNext() {
+        print(nextPage)
+        print(isFetching)
+        if let nextPage = self.nextPage, !isFetching {
+            activityIndicator.isHidden = false;
+            activityIndicator.startAnimating();
+            isFetching = true;
+            AF.request(nextPage, method: .get).response { response in self.doneFetchingNext(data: response.data, response: response.response, error: response.error)}
```

Comment:
Is there any reason why `fetchNext` uses Alamorfire, but the initial `fetch` uses some other fetch library?

Should it be also chaged to use Alamofire for consistency?